orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -82,6 +82,10 @@ class DependentPipelineStarter implements ApplicationContextAware {
 
     def pipeline
 
+    if (pipelineConfig.disabled) {
```

Comment:
Can this check go closer to the top of this function ... at a glance it seems like it could.

Reply:
yeah, it can, good call